dsl/src/main/groovy/org/cristalise/dsl/persistency/outcome/DynamicForms.groovy

```diff
@@ -29,6 +29,7 @@ import groovy.transform.CompileStatic;
 
 @CompileStatic
 class DynamicForms {
+    String acceptMimeTypes = null
```

Comment:
@kovax This name is too specific. The standard HTML attribute is called `accept` because one can specify not only mime types, but also file extensions, see https://www.w3schools.com/tags/att_input_accept.asp . Therefore, I would rather use the standard html attribute name.